CougSat1-ADCS/src/drivers/M24M01R.h

```diff
@@ -0,0 +1,47 @@
+/******************************************************************************
+ * Copyright (c) 2018 by Cougs in Space - Washington State University         *
+ * Cougs in Space website: cis.vcea.wsu.edu                                   *
+ *                                                                            *
+ * This file is a part of flight and/or ground software for Cougs in Space's  *
+ * satellites. This file is proprietary and confidential.                     *
+ * Unauthorized copying of this file, via any medium is strictly prohibited.  *
+ ******************************************************************************/
+/**
+ * @file M24M01R.h
+ * @author Eric Curtland
+ * @brief 
+ * @date 2019-03-24
+ * 
+ */
+#ifndef M24M01R_H_
+#define M24M01R_H_
+
+#define M24M01_PAGE_SIZE 256
+#define DEVICE_TYPE_ID 0x10
+#define MODE_WRITE 0
```

Comment:
Don't need to set the address to read and write because the library will automatically assert the read and write bits.